src/App.jsx

```diff
@@ -7,11 +7,12 @@ import AddTodoForm from "./AddTodoForm";
 
 function App() {
   // Define the todo list array
-
+  const [newTodo, setNewTodo] = useState("");
   return (
     <div>
       <h1>Todo List</h1>
-      <AddTodoForm />
+      <AddTodoForm onAddTodo={setNewTodo}/>
```

Comment:
You’re passing `setNewTodo` directly to `AddTodoForm`. While this works, it would be cleaner to introduce a handler function for adding new todos.
[Why you shouldn’t pass React’s setState as a prop: a deep dive](https://medium.com/@christopherthai/why-you-shouldnt-pass-react-s-setstate-as-a-prop-a-deep-dive-8a3dcd74bec8)
 [Is it OK to pass setState as a prop in React?](https://tigerabrodi.blog/is-it-ok-to-pass-setstate-as-a-prop-in-react#)